Tools/cases_generator/analysis.py

```diff
@@ -381,7 +381,8 @@ def analyze_pseudo(self, pseudo: parsing.Pseudo) -> PseudoInstruction:
         # Make sure the targets have the same fmt
         fmts = list(set([t.instr_fmt for t in targets]))
         assert len(fmts) == 1
-        assert len(list(set([t.instr_flags.bitmap() for t in targets]))) == 1
+        ignored_flags = set(('HAS_EVAL_BREAK_FLAG',))
+        assert len(list(set([t.instr_flags.bitmap(ignore=ignored_flags) for t in targets]))) == 1
```

Comment:
Use set and set comprehension notation:
```suggestion
        ignored_flags = {'HAS_EVAL_BREAK_FLAG'}
        assert len({t.instr_flags.bitmap(ignore=ignored_flags) for t in targets}) == 1
```